### PR TITLE
Added documentation for RUM APM correlation

### PR DESCRIPTION
**Pull request by hdelaby**
### What does this PR do?

Add a new documentation page for connecting RUM and APM.

### Motivation

New feature (still in **beta** for now)

### Preview

https://docs-staging.datadoghq.com/hdelaby/connect-rum-apm/real_user_monitoring/connect_rum_and_traces/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/connect_rum_and_traces/_index.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    text: "APM and Distributed Tracing"
+---
+
+{{< img src="tracing/connect_rum_and_traces/rum_trace_tab.png" alt="RUM and Traces"  style="width:100%;">}}
+
+# Overview
+
+The APM integration with Real User Monitoring allows you to link requests from your web and mobile applications to their corresponding backend traces, while providing the RUM session context.
+
+Having frontend-related specifics (thanks to your RUM session) as well as backend, infrastructure, and log information (thanks to your trace id injection) allows you to pinpoint issues anywhere in your stack in unprecedented speed with a full understanding of what your users are experiencing.
```
Comment (apigirl) [inline content/en/tracing/connect_rum_and_traces/_index.md:19]:
```suggestion
Use frontend data from RUM, and backend, infrastructure, and log information from trace ID injection to quickly pinpoint issues anywhere in your stack and fully understand what your users are experiencing.
```

File: content/en/tracing/connect_rum_and_traces/_index.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+<div class="alert alert-warning">
+This feature is currently available for private beta. <a href="https://docs.datadoghq.com/help/">Reach out to support</a> to turn on this feature for your account.
+</div>
+
+# Usage
+## Prerequisites
+
+-   Your services targeted by your RUM applications are [traced on the APM side][1].
```
Comment (apigirl) [inline content/en/tracing/connect_rum_and_traces/_index.md:28]:
```suggestion
-   Set up [APM tracing][1] on the services targeted by your RUM applications.
```

File: content/en/tracing/connect_rum_and_traces/_index.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+-   Your services use an HTTP server.
+-   Your HTTP servers are using a library that supports distributed tracing.
```
Comment (apigirl) [inline content/en/tracing/connect_rum_and_traces/_index.md:30]: do we have a list of these that we can link to? I am not really sure what you mean here

File: content/en/tracing/connect_rum_and_traces/_index.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+## RUM set up
+{{< tabs >}}
+{{% tab "Browser RUM" %}}
+
+1.  Set up [Browser Real User Monitoring][1]
```
Comment (apigirl) [inline content/en/tracing/connect_rum_and_traces/_index.md:36]:
```suggestion
1.  Set up [Browser Real User Monitoring][1].
```

File: content/en/tracing/connect_rum_and_traces/_index.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+})
+```
+
+**Note**: `allowedTracingOrigins` accepts Javascript String and RegExp.
+
+[1]: /real_user_monitoring/browser/
+{{% /tab %}}
+{{% tab "Android RUM" %}}
+
+1.  Set up [Android Real User Monitoring][1]
```
Comment (apigirl) [inline content/en/tracing/connect_rum_and_traces/_index.md:57]:
```suggestion
1.  Set up [Android Real User Monitoring][1].
```

File: content/en/tracing/connect_rum_and_traces/_index.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+    
+2.  Configure the OkHttpClient interceptor with the list of internal (first party) origins called by your Android application.
```
Comment (apigirl) [inline content/en/tracing/connect_rum_and_traces/_index.md:59]:
```suggestion
2.  Configure the `OkHttpClient` interceptor with the list of internal (first party) origins called by your Android application.
```

File: content/en/tracing/connect_rum_and_traces/_index.md [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+val tracedHosts =  listOf("example.com", "example.eu")
+
+val okHttpClient = OkHttpClient.Builder()
+    .addInterceptor(DatadogInterceptor(tracedHosts))
+    .addNetworkInterceptor(TracingInterceptor(tracedHosts))
+    .eventListenerFactory(DatadogEventListener.Factory())
+    .build()
+```
+
+**Note**: By default, all subdomains of listed hosts will be traced. For instance, adding example.com will also enable the tracing for api.example.com and foo.example.com
```
Comment (apigirl) [inline content/en/tracing/connect_rum_and_traces/_index.md:70]:
```suggestion
**Note**: By default, all subdomains of listed hosts are traced. For instance, if you add `example.com`, you also enable the tracing for `api.example.com` and `foo.example.com`.
```

File: content/en/tracing/connect_rum_and_traces/_index.md [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+
+[1]: /real_user_monitoring/android/
+{{% /tab %}}
+{{% tab "iOS RUM" %}}
+
+1.  Set up [iOS Real User Monitoring][1]
```
Comment (apigirl) [inline content/en/tracing/connect_rum_and_traces/_index.md:76]:
```suggestion
1.  Set up [iOS Real User Monitoring][1].
```

File: content/en/tracing/connect_rum_and_traces/_index.md [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+3.  Initialize URLSession as stated in the [set up documentation][1]:
+```swift
+let session =  URLSession(
+    configuration: ...,
+    delegate: DDURLSessionDelegate(),
+    delegateQueue: ...
+)
+```
+
+**Note**: By default, all subdomains of listed hosts will be traced. For instance, adding example.com will also enable the tracing for api.example.com and foo.example.com
```
Comment (apigirl) [inline content/en/tracing/connect_rum_and_traces/_index.md:98]:
```suggestion
**Note**: By default, all subdomains of listed hosts are traced. For instance, if you add `example.com`, you also enable the tracing for `api.example.com` and `foo.example.com`.
```

**Review by apigirl — COMMENTED** [review]
some small feedbacks! this is very good. exciting functionality!

File: content/en/tracing/connect_rum_and_traces/_index.md (hunk 3 above)
Comment (hdelaby) [reply content/en/tracing/connect_rum_and_traces/_index.md:30]: Should we link to the section underneath?

Comment (apigirl) [reply content/en/tracing/connect_rum_and_traces/_index.md]: yes, perfect

**Review by andrewardito — APPROVED** [review]
LGTM!

File: content/en/real_user_monitoring/connect_rum_and_traces/_index.md [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+<div class="alert alert-warning">
+This feature is currently available for private beta. <a href="https://docs.datadoghq.com/help/">Reach out to support</a> to turn on this feature for your account.
+</div>
+
+# Usage
+## Prerequisites
+
+-   Set up [APM tracing][1] on the services targeted by your RUM applications.
+-   Your services use an HTTP server.
+-   Your HTTP servers are using a library that supports distributed tracing.
```
Comment (apigirl) [inline content/en/real_user_monitoring/connect_rum_and_traces/_index.md:30]:
```suggestion
-   Your HTTP servers are using [a library that supports distributed tracing](#supported-libraries).
```

File: content/en/real_user_monitoring/connect_rum_and_traces/_index.md [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+)
+```
+
+**Note**: By default, all subdomains of listed hosts are traced. For instance, if you add `example.com`, you also enable the tracing for `api.example.com` and `foo.example.com`.
+
+[1]: /real_user_monitoring/ios/
+{{% /tab %}}
+{{< /tabs >}}
+
+# Supported Libraries
```
Comment (apigirl) [inline content/en/real_user_monitoring/connect_rum_and_traces/_index.md:104]:
```suggestion
## Supported Libraries
```

File: content/en/real_user_monitoring/connect_rum_and_traces/_index.md [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+    tag: "Blog"
+    text: "Real User Monitoring"
+  - link: "/tracing/"
+    tag: "Documentation"
+    text: "APM and Distributed Tracing"
+---
+
+{{< img src="real_user_monitoring/connect_rum_and_traces/rum_trace_tab.png" alt="RUM and Traces"  style="width:100%;">}}
+
+# Overview
```
Comment (apigirl) [inline content/en/real_user_monitoring/connect_rum_and_traces/_index.md:15]:
```suggestion
```

File: content/en/real_user_monitoring/connect_rum_and_traces/_index.md [hunk 13]
```diff
@@ -0,0 +1,155 @@
...
+
+The APM integration with Real User Monitoring allows you to link requests from your web and mobile applications to their corresponding backend traces. This combination lets you see your full front end and backend data through one lens.
+
+Use frontend data from RUM, and backend, infrastructure, and log information from trace ID injection to quickly pinpoint issues anywhere in your stack and fully understand what your users are experiencing.
+
+<div class="alert alert-warning">
+This feature is currently available for private beta. <a href="https://docs.datadoghq.com/help/">Reach out to support</a> to turn on this feature for your account.
+</div>
+
+# Usage
```
Comment (apigirl) [inline content/en/real_user_monitoring/connect_rum_and_traces/_index.md:25]:
```suggestion
## Usage
```

File: content/en/real_user_monitoring/connect_rum_and_traces/_index.md [hunk 14]
```diff
@@ -0,0 +1,155 @@
...
+## Prerequisites
+
+-   Set up [APM tracing][1] on the services targeted by your RUM applications.
+-   Your services use an HTTP server.
+-   Your HTTP servers are using a library that supports distributed tracing.
+
+## RUM set up
```
Comment (apigirl) [inline content/en/real_user_monitoring/connect_rum_and_traces/_index.md:32]:
```suggestion
### RUM set up
```

File: content/en/real_user_monitoring/connect_rum_and_traces/_index.md [hunk 15]
```diff
@@ -0,0 +1,154 @@
...
+The APM integration with Real User Monitoring allows you to link requests from your web and mobile applications to their corresponding backend traces. This combination lets you see your full front end and backend data through one lens.
+
+Use frontend data from RUM, and backend, infrastructure, and log information from trace ID injection to quickly pinpoint issues anywhere in your stack and fully understand what your users are experiencing.
+
+<div class="alert alert-warning">
+This feature is currently available for private beta. <a href="https://docs.datadoghq.com/help/">Reach out to support</a> to turn on this feature for your account.
+</div>
+
+## Usage
+## Prerequisites
```
Comment (apigirl) [inline content/en/real_user_monitoring/connect_rum_and_traces/_index.md:25]:
```suggestion
### Prerequisites
```

**Review by apigirl — APPROVED** [review]

**Pull request merged**